scripts/setup.sh

```diff
@@ -30,20 +30,19 @@ function setup_client() {
 }
 
 function create_db(){
-	if psql -lqtA | cut -d\| -f1 | grep -qxF "a_socials"; then
+    echo "Creating user a_socials"
+    PGPASSWORD=postgres createuser -U postgres -w -P -s -e a_socials
+	echo "Successfully created user a_socials"
+    if psql -U a_socials -lqtA | cut -d\| -f1 | grep -qxF "a_socials"; then
 		echo "You already have a database named a_socials."
 	else
 		echo "You need to create a database."
 		echo -e "\n\n\033[31mPlease provide a single argument for the username/dbname\033[0m\n"
 		read dbname
 		DBNAME=$dbname
-		echo -e "\n\n\033[31mPlease provide just one argument as the username\033[0m\n"
-		read dbuser
-		USERNAME_DBNAME=$dbuser
-		if ! psql -lqtA | grep  $USERNAME_DBNAME; then
```

Comment:
Do you know what this line actually takes care of

Reply:
That line is checking if the database already exists. However, the problem with the previous code is that the user to connect to when running the command was not specified leading the machine to use the logged in user as the user role which led to complication during authentication. The only change I did here to circumvent this complication was to use a created user role with its appropriate password to run the command